template/bison/yacc.c

```diff
@@ -1740,4 +1742,4 @@ YYLTYPE yylloc = yyloc_default;
 <%# b4_percent_code_get([[epilogue]]) -%>
 #line <%= output.aux.epilogue_first_lineno - 1 %> "<%= output.grammar_file_path %>"
 
-<%= output.aux.epilogue -%>
+<%= output.aux.epilogue -%>
```

Comment:
Could you add a newline here? There is a newline on the end of this file in ruby/ruby (e1ccb2838b65f23e7a21bdd416160d03dd73e7f8).

```diff
--- a/ruby/ruby/tool/lrama/template/bison/yacc.c
+++ b/template/bison/yacc.c
@@ -1742,4 +1742,4 @@ yyreturnlab:
 <%# b4_percent_code_get([[epilogue]]) -%>
 #line <%= output.aux.epilogue_first_lineno - 1 %> "<%= output.grammar_file_path %>"

-<%= output.aux.epilogue -%>
+<%= output.aux.epilogue -%>
\ No newline at end of file
```

Reply:
@yui-knk Sure! I added a newline and fixup in [c705968](https://github.com/ruby/lrama/pull/40/commits/c705968292968e0fd62566954424f9c21f84686c)